### ISSUE TITLE
Write a better readme

### ISSUE THREAD

**Issue by yet-another-lucas** [yet-another-lucas/mustadio#5]
The readme is very very low effort content.

**Comment by sagarchoudhary96** [comment]
@yet-another-lucas I would like to contribute to this.

**Comment by yet-another-lucas** [comment]
@sagarchoudhary96 Great to have your help. Feel free to play around with the code and see if you can think up a better readme that captures what it does and how to use it.

**Comment by sagarchoudhary96** [comment]
definitely👍🏻

**Comment by GeekyShiva** [comment]
it's really nice exemplary project to work on ...... @sagarchoudhary96 , are you in process of making a PR???

**Comment by sagarchoudhary96** [comment]
@GeekyShiva yep i would like to can you let me know the steps to run or how to use this project?

**Comment by GeekyShiva** [comment]
@yet-another-lucas is a better person to answer this
